content/en/platform/corda/5.2/key-concepts/cluster-admin/_index.md

```diff
@@ -96,10 +96,9 @@ All keys are stored in the Crypto databases (cluster and virtual nodes) and they
 	 figcaption="Wrapping Keys"
 >}}
 
-The diagram illustrates that key wrapping is hierarchical. The master wrapping key protects other wrapping keys, such as the virtual node related keys, which in turn protect the private keys used by Corda. It must never be possible for someone with a copy of the Corda database, or a virtual node database, to decrypt the keys stored in the database using other information stored in the database. Therefore, the master wrapping key, or the information required to generate this key, must be stored and managed outside Corda. This can be achieved in one of the following ways:
+The diagram illustrates that key wrapping is hierarchical. The master wrapping key protects other wrapping keys, such as the virtual node related keys, which in turn protect the private keys used by Corda. The master wrapping key, or the information required to generate this key, must be stored and managed outside Corda. This ensures that it is not possible to decrypt the keys stored in the database using other information stored in the same database. By default, the Corda default secrets lookup service generates the master key using a passphrase and salt set for the crypto worker processes. {{< enterprise-icon noMargin="true" >}}Alternatively, Corda Enterprise can retrieve the master key from an external key management system that Corda retrieves when required.
```

Comment:
"The master wrapping key, or the information required to generate this key..." see comment above, I'm not sure this is quite explaining the situation. Production systems would manage the key externally, non-production can use the built in default secrets service.